src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -686,7 +686,7 @@ DropMetadataSnapshotOnNode(WorkerNode *workerNode)
 	bool singleTransaction = true;
 	List *dropMetadataCommandList = DetachPartitionCommandList();
 	dropMetadataCommandList = lappend(dropMetadataCommandList,
-									  BREAK_CITUS_TABLE_SEQUENCE_DEPENDENCY_COMMAND);
+									  BREAK_ALL_CITUS_TABLE_SEQUENCE_DEPENDENCY_COMMAND);
```

Comment:
If we are going to drop sequences per table, what is the use of keeping this?

Reply:
I think we can get rid of them, as you pointed out, since now we break sequence dependencies for each table before creating it.

Reply:
makes sense